digitalocean/app/app_spec.go

```diff
@@ -186,6 +191,31 @@ func appSpecDomainSchema() *schema.Resource {
 	}
 }
 
+func appSpecVPCSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The ID of the VPC droplet.",
+			},
+			"egress_ips": {
```

Comment:
Egress ips are read only. This value will come from the response but not when we are creating it. Did you by any chance run an existing tests with a tf file that contains `vpc { egress_ips: [] }` ? If not, I think it'd break.

If you did, I wonder why it didn't break!